hack/lib/tracing.bash

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+function install_tracing {
+  deploy_zipkin
+  enable_eventing_tracing
+}
+
+function deploy_zipkin {
```

Comment:
Shouldn't we use Openshift Tracing (Jaeger) here?

Reply:
No idea. Just trying to make things work honestly.
Maybe we can do that later

Reply:
I will create a JIRA for it after this PR is merged